src/main/kotlin/org/taktik/icure/domain/be/kmehr/IncapacityExportInfo.kt

```diff
@@ -39,4 +39,6 @@ data class IncapacityExportInfo(
 	val recoveryAddress: Address? = null,
 	val foreignStayBegin: Long = 0, //MMEDIATT-ITEM foreignstay
 	val foreignStayEnd: Long = 0,
+	val softwareName: String? = null,
```

Comment:
It is weird to set it at "" when you use null to fallback to the default values. That means that the default values will never be used.